controllers/controllers.go

```diff
@@ -149,11 +149,7 @@ func (r *instanceReconciler) deconfigureInstance(node *core.Node) error {
 func windowsNodePredicate(byoh bool) predicate.Funcs {
 	return predicate.Funcs{
 		CreateFunc: func(e event.CreateEvent) bool {
-			if e.Object.GetLabels()[core.LabelOSStable] != "windows" {
-				return false
-			}
-			if (byoh && e.Object.GetLabels()[BYOHLabel] != "true") ||
-				(!byoh && e.Object.GetLabels()[BYOHLabel] == "true") {
```

Comment:
Ideally the refactor should have been a separate commit but the delta is small enough that I am willing to give it a pass. However please add a line in the commit message saying you are introducing a helper.